cardano-cli/test/Test/CLI/Shelley/Golden/StakeAddress/DeregistrationCertificate.hs

```diff
@@ -5,25 +5,23 @@ module Test.CLI.Shelley.Golden.StakeAddress.DeregistrationCertificate
   ) where
 
 import           Cardano.Prelude
-
 import           Hedgehog (Property)
 
 import qualified Test.OptParse as OP
 
 {- HLINT ignore "Use camelCase" -}
 
 golden_shelleyStakeAddressDeregistrationCertificate :: Property
-golden_shelleyStakeAddressDeregistrationCertificate = OP.propertyOnce $ do
-  OP.moduleWorkspace "tmp" $ \tempDir -> do
-    verificationKeyFile <- OP.noteInputFile "test/Test/golden/shelley/keys/stake_keys/verification_key"
-    deregistrationCertFile <- OP.noteTempFile tempDir "deregistrationCertFile"
+golden_shelleyStakeAddressDeregistrationCertificate = OP.propertyOnce . OP.moduleWorkspace "tmp" $ \tempDir -> do
+  verificationKeyFile <- OP.noteInputFile "test/Test/golden/shelley/keys/stake_keys/verification_key"
+  deregistrationCertFile <- OP.noteTempFile tempDir "deregistrationCertFile"
 
-    void $ OP.execCardanoCLI
-        [ "shelley","stake-address","deregistration-certificate"
-        , "--staking-verification-key-file", verificationKeyFile
-        , "--out-file", deregistrationCertFile
-        ]
+  void $ OP.execCardanoCLI
+      [ "shelley","stake-address","deregistration-certificate"
```

Comment:
Isn't SH supposed to enforce a 2 space indent?

Reply:
Seems like SH isn't enforcing this.